org.jacoco.doc/docroot/doc/repo.html

```diff
@@ -48,41 +48,71 @@
 <h2>Artifacts</h2>
 
 <p>
-  JaCoCo is split into several separate projects, the artifact ids are:   
+  Following JAR files are available:
 </p>
 
 <table class="coverage">
   <thead>
     <tr>
       <td>Group ID</td>
       <td>Artifact ID</td>
+      <td>Classifier</td>
       <td>Description</td>
     </tr>
   </thead>
   <tbody>
     <tr>
       <td><code>org.jacoco</code></td>
       <td><code>jacoco-maven-plugin</code></td>
-      <td>Plug-in for Maven builds</td>
+      <td></td>
+      <td>Plug-in for Maven</td>
     </tr>
     <tr>
       <td><code>org.jacoco</code></td>
       <td><code>org.jacoco.agent</code></td>
+      <td></td>
       <td>API to get a local copy of the agent</td>
     </tr>
     <tr>
       <td><code>org.jacoco</code></td>
+      <td><code>org.jacoco.agent</code></td>
+      <td><code>runtime</code></td>
+      <td>Agent</td>
+    </tr>
+    <tr>
+      <td><code>org.jacoco</code></td>
       <td><code>org.jacoco.ant</code></td>
+      <td></td>
       <td>Ant Tasks</td>
     </tr>
     <tr>
       <td><code>org.jacoco</code></td>
+      <td><code>org.jacoco.ant</code></td>
+      <td><code>nodeps</code></td>
+      <td>Ant Tasks <i>(all dependencies included)</i></td>
+    </tr>
+    <tr>
+      <td><code>org.jacoco</code></td>
+      <td><code>org.jacoco.cli</code></td>
+      <td></td>
+      <td>Command Line Interface</td>
+    </tr>
+    <tr>
+      <td><code>org.jacoco</code></td>
+      <td><code>org.jacoco.cli</code></td>
+      <td><code>nodeps</code></td>
```

Comment:
@marchof 
Answering on https://github.com/jacoco/jacoco/pull/525#discussion_r117828935 : IMO `Command Line Interface` in Maven Repository can be presented as a single artifact with all dependencies included, i.e. replace current artifact without classifier by artifact with classifier `nodeps` - it is nor OSGi Bundle, nor provides any APIs and this will be consistent with how it is presented in our ZIP distribution. WDYT?